app/controllers/evidence_controller.rb

```diff
@@ -74,5 +78,9 @@ def evidence_result
     @result = Evidence::Views::Result.new(@evidence)
   end
 
+  def evidence_confirmation
+    @confirmation = EvidenceCheck.find(params[:id])
```

Comment:
Just a little thing, I'd call this `@evidence_check` rather than `@confirmation`, so it's clear in the template what the object represents.